components/badge.css

```diff
@@ -0,0 +1,111 @@
+:root {
+}
+.badge-section {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+.badge-section h1 {
+  font-size: 48px;
+  margin-top: 2rem;
+}
+.badge-section h4 {
+  font-size: 24px;
+  margin-bottom: 1rem;
+  font-weight: 500;
+}
+.badges-avatar {
+  display: flex;
+  align-items: center;
+}
+.badges-avatar img {
+  margin-right: 4rem;
+}
+.lg-badge-container {
+  position: relative;
+}
+.lg-badge-container .badgr-lg {
```

Comment:
I think it should be .badge-lg (spelling mistake)

Reply:
No, it's correct but I will write a more specific classname.